### DISCUSSION TITLE
Manually set game proxy

### DISCUSSION THREAD

**Discussion by acha666** [ppy/osu#25215] [Q&A] [answered]
Hey there,

I'm dealing with network censorship in China, so I rely on a proxy server to ensure a stable connection for my game.

It appears that the game automatically detects and uses the system's proxy settings when it starts up. However, changing the settings doesn't have any effect on the running game until I manually restart it.

I'm wondering if there's a way to manually set the proxy for the game, overriding the system proxy settings. Maybe there's a command-line parameter I can use?

**Comment by BlockHead007** [comment, marked as answer]
#25008
